### ISSUE TITLE
v1.1.1 reports itself as v1.1.0

### ISSUE THREAD

**Issue by kevbo** [pyenv/pyenv#947]
I just brew upgraded to v1.1.1, but then noticed that it was still reporting itself as v1.1.0. I did a `brew unlink` and `brew link` to recreate the symlinks, which didn't fix the issue. Then I looked at the source in the installed directory:

```
kboers-mbp:libexec kboers $ pwd
/usr/local/Cellar/pyenv/1.1.1/libexec
kboers-mbp:libexec kboers $ grep version= pyenv---version
version="1.1.0"
```

**Comment by blueyed** [comment]
ping @Samureus

**Comment by yyuu** [comment]
I confirmed the version is 1.1.0 in v1.1.1 https://github.com/pyenv/pyenv/blob/v1.1.1/libexec/pyenv---version#L15

It'd be sufficient to cut a new release from current master. Will do.

**Comment by yyuu** [comment]
I've released v1.1.2 in favor of fixing v1.1.1. Please let me know if you somehow want me to fix v1.1.1 itself. So that I will create another hotfix for it.

https://github.com/pyenv/pyenv/releases/tag/v1.1.2
